gdu: add page (#6447)
tldr-pages/tldr: pages/common/gdu.md

# gdu

> Disk usage analyzer with console interface.
> More information: <https://github.com/dundee/gdu>.

> - Show disk usage of current directory:

Reviewer: ```suggestion
- Show disk usage of the current directory:
```

> - Show disk usage of the current directory:

Reviewer 2: ```suggestion
- Show the disk usage of the current directory:
```

`gdu`

> - Show disk usage of given directory:

Reviewer 2: ```suggestion
- Show the disk usage of a given directory:
```

> `gdu {{path/to/dir}}`

Reviewer: ```suggestion
`gdu {{path/to/directory}}`
```

> - Show disk usage of all mounted disks:

Reviewer 2: ```suggestion
- Show the disk usage of all mounted disks:
```

`gdu --show-disks`

> - Show disk usage but ignore some paths:
> 
> `gdu --ignore-dirs {{/sys,/proc}} /`

Reviewer 2: If this page is under common, we sholudn't use Linux-specefic directories. Also, it would be easier if we remove the second argument.
```suggestion
- Show the disk usage of the current directory but ignore some sub-directories:

`gdu --ignore-dirs {{path/to/directory1,path/to/directory2,...}}`
```
You can of course change something if you like to.

> - Ignore paths by regular pattern:

Reviewer 2: Not sure if there's a difference between `regular pattern` and `regular expression`, but the latter is definitively more commmon.
```suggestion
- Ignore paths by regular expression:
```

> `gdu --ignore-dirs-pattern {{'.*[abc]+'}}`

Reviewer 2: ```suggestion
`gdu --ignore-dirs-pattern '{{.*[abc]+}}'`
```

> `gdu --ignore-diRS {{path/to/directory1,path/to/directory2,...}}`

Reviewer 3: ```suggestion
`gdu --ignore-dirs {{path/to/directory1,path/to/directory2,...}}`
```

- Ignore hidden directories:

`gdu --no-hidden`

> - Only print result, do not enter interactive mode:

Reviewer 2: So all the other examples are interactive? In which way is it interactive?

Reviewer 4: I guess it's like `fdisk` interactive mode.

Reviewer 2: Then we should change them all to `Interactively show the disk usage ...`.

> - Only print result, do not enter interactive mode:

Reviewer 2: ```suggestion
- Only print the result, do not enter interactive mode:
```

`gdu --non-interactive {{path/to/dir}}`

> - Do not show progress in non-interactive mode, useful in script:

Reviewer 5: ```suggestion
- Do not show progress in non-interactive mode, useful in scripts:
```
❓

> - Do not show progress in non-interactive mode, useful in scripts:

Reviewer 2: ```suggestion
- Do not show progress in non-interactive mode (useful in scripts):
```

> - Do not show progress in non-interactive mode (useful in scripts):

Reviewer 2: ```suggestion
- Do not show the progress in non-interactive mode (useful in scripts):
```
